src/bentoml/_internal/frameworks/diffusers.py

```diff
@@ -525,7 +498,7 @@ def import_model(
             )
 
     elif pipeline_class:
-        _try_import_huggingface_hub()
+        is_huggingface_hub_available()
```

Comment:
Why do we need to check for huggingface_hub available in this branch here?

Reply:
This is diffusers which is something Larme worked on so I'm not entirely sure. But a couple lines down, `from huggingface_hub.file_download import REGEX_COMMIT_HASH` seems to use huggingface hub.